src/storage/txn/flow_controller/singleton_flow_controller.rs

```diff
@@ -126,22 +128,21 @@ impl EngineFlowController {
                 .build(),
         );
         let discard_ratio = Arc::new(AtomicU32::new(0));
-        let checker = FlowChecker::new(config, engine, discard_ratio.clone(), limiter.clone());
+        let config_tracker = Arc::new(VersionTrack::new(config.clone()));
```

Comment:
Just trying to learn, what's the benefit of using a VersionTracker instead of a RwLock? Do we use the version anywhere?

Reply:
There is no much difference here as we don't have any `Tracker` of this config.